src/main/java/com/minecolonies/coremod/colony/requestsystem/requests/StandardRequestFactories.java

```diff
@@ -0,0 +1,259 @@
+package com.minecolonies.coremod.colony.requestsystem.requests;
+
+import com.google.common.reflect.TypeToken;
+import com.minecolonies.api.colony.requestsystem.RequestState;
+import com.minecolonies.api.colony.requestsystem.factory.IFactoryController;
+import com.minecolonies.api.colony.requestsystem.request.IRequestFactory;
+import com.minecolonies.api.colony.requestsystem.requestable.Delivery;
+import com.minecolonies.api.colony.requestsystem.requester.IRequester;
+import com.minecolonies.api.colony.requestsystem.token.IToken;
+import net.minecraft.item.ItemStack;
+import net.minecraft.nbt.NBTTagCompound;
+import net.minecraft.nbt.NBTTagInt;
+import net.minecraft.nbt.NBTTagList;
+import net.minecraftforge.common.util.Constants;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * Final class holding all the factories for requestables inside minecolonie
+ */
+public final class StandardRequestFactories
+{
+
+    ////// --------------------------- NBTConstants --------------------------- \\\\\\
+    private static final String NBT_TOKEN     = "Token";
+    private static final String NBT_STATE     = "State";
+    private static final String NBT_REQUESTED = "Requested";
+    private static final String NBT_RESULT    = "Result";
+    private static final String NBT_PARENT    = "Parent";
+    private static final String NBT_CHILDREN  = "Children";
+    ////// --------------------------- NBTConstants --------------------------- \\\\\\
+
+    public static final class ItemStackFactory implements IRequestFactory<ItemStack, StandardRequests.ItemStackRequest>
+    {
+        /**
+         * Method to get a new instance of a request given the input and token.
+         *
+         * @param input        The input to build a new request for.
+         * @param location     The location of the requester.
+         * @param token        The token to build the request from.
+         * @param initialState The initial state of the request request.
+         * @return The new output instance for a given input.
+         */
+        @Override
+        public StandardRequests.ItemStackRequest getNewInstance(@NotNull ItemStack input, @NotNull IRequester location, @NotNull IToken token, @NotNull RequestState initialState)
+        {
+            return new StandardRequests.ItemStackRequest(location, token, initialState, input);
+        }
+
+        @NotNull
+        @Override
+        public TypeToken<StandardRequests.ItemStackRequest> getFactoryOutputType()
+        {
+            return new TypeToken<StandardRequests.ItemStackRequest>() {};
+        }
+
+        @NotNull
+        @Override
+        public TypeToken<ItemStack> getFactoryInputType()
+        {
+            return new TypeToken<ItemStack>() {};
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Move this left curly brace to the beginning of next line of code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)